pkg/plugin/metrics.go

```diff
@@ -0,0 +1,25 @@
+package plugin
+
+import "github.com/prometheus/client_golang/prometheus"
+
+func init() {
+	registerPrometheusMetrics()
+}
+
+func registerPrometheusMetrics() {
+	prometheus.MustRegister(kmsOperationCounter)
+}
+
+var (
+	kmsOperationCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "aws_encryption_provider_operations_total",
```

Comment:
nit: could you rename this to `aws_encryption_provider_kms_operations_total`? It just clarifies that this metric is tracking `kms` metrics

Reply:
sure thing. done